### ISSUE TITLE
Claim package name modal does not have enough contrast

### ISSUE THREAD

**Issue by shuuji3** [npmx-dev/npmx.dev#1872]
### Describe the bug

The warning alert text color is too light and it's difficult to read on the light theme.

I saw a few different alert stylings in the app. Maybe we should create a common alert component with proper color contrasts.

<img width="1091" height="814" alt="screenshot of claim package name modal, 'Consider using a scoped package instead' etc has low contract on light theme" src="https://github.com/user-attachments/assets/9f66d22d-3b89-429f-bdba-4b919e2d1c94" />

example: https://npmx.dev/search?q=npmx-atproto

### Logs

```shell-script

```
